### PR TITLE
Client SDK for MarketplaceOrdering Get and Create Agreement Terms APIs

### PR DESCRIPTION
**Pull request by prbansa**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [*] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [*] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [*] Title of the pull request is clear and informative.
- [*] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [*] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [*] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.MarketplaceOrdering</PackageId>
+    <Description>Provides developers with a library to get and set legal terms for Azure Marketplace Offers. Note: This client library is for MarketplaceOrdering under Azure Resource Manager.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.Management.MarketplaceOrdering</AssemblyName>
+    <PackageTags>Microsoft Azure MarketplaceOrdering Management;REST HTTP client;windowsazureofficial;netcore451511</PackageTags>
```
Comment (shahabhijeet) [inline src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj]: Add < PackageReleaseNotes > and add Release notes.
I would recommend to use < ![CDATA and then format your Release notes

File: src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.MarketplaceOrdering</PackageId>
+    <Description>Provides developers with a library to get and set legal terms for Azure Marketplace Offers. Note: This client library is for MarketplaceOrdering under Azure Resource Manager.</Description>
+    <VersionPrefix>1.0.0</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj]: Also as this is your first version, is your service GA already? If not then mark your version as 1.0.0-preview

File: src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="SessionRecords\MarketplaceOrdering.Tests.AgreementsTests\**" />
+    <EmbeddedResource Remove="SessionRecords\MarketplaceOrdering.Tests.AgreementsTests\**" />
+    <None Remove="SessionRecords\MarketplaceOrdering.Tests.AgreementsTests\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- <PackageReference Include="Microsoft.Azure.Management.MarketplaceOrdering" Version="1.0.0" /> -->
```
Comment (shahabhijeet) [inline src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj]: Remove commented code

File: src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <PackageId>MarketplaceOrdering.Tests</PackageId>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <Description>MarketplaceOrdering.Tests Class Library</Description>
+    <Authors>Microsoft Corporation</Authors>
+    <AssemblyName>MarketplaceOrdering.Tests</AssemblyName>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
```
Comment (shahabhijeet) [inline src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj]: Any reasons you need this ItemGroup? If none, Please remove

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj (hunk 2 above)
Comment (prbansa) [reply src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj]: REST API is already in prod, the swagger is being merged into public repo.

File: src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj (hunk 1 above)
Comment (prbansa) [reply src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj]: Updated. 👍

File: src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj (hunk 3 above)
Comment (prbansa) [reply src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj]: Updated

File: src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj (hunk 4 above)
Comment (prbansa) [reply src/SDKs/MarketplaceOrdering/MarketplaceOrdering.Tests/MarketplaceOrdering.Tests.csproj]: There is no reason, deleting this.

**Comment by prbansa** [comment]
@shahabhijeet  please review my PR after updates as per your suggestions.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**